assets/scss/templates/_reportes.scss

```diff
@@ -0,0 +1,44 @@
+.hero-filters {
```

Comment:
Why this name? `.hero-filters` wouldn't it be better `.report-filters`? I'm asking out of curiosity.

Reply:
I like the name because this same style could be used for other areas than reports.

It is not consistent with the filename:  `_reportes.scss`, but it could be improved later.

Reply:
@cbernat normalmente ese espacio en el layout se lo llama hero-banner, sirve para hacer promociones, o meter un carousel de promos por eso puse ese nombre. Pero viendo tu comentario (muy acertado), nosotros no tenemos ni promociones ni descuentos de viajes, así que me parece muy bien cambiarle el nombre :) Thanks @cbernat!

Reply:
@andresmoschini como bien decis podemos mejorarlo mas adelante! ;)

Reply:
Creo que comenté en el commit, entonces quedó fuera de la conversación.

No tengo problema en dejarle `report-filter`, pero que quede constancia que me gustaba más el otro :P